### PR TITLE
WIP Adding posgres to docker compose config, add as service to API pr…

### PR DESCRIPTION
**Pull request by mikelax**
…oject.

### CODE REVIEW

**Comment by mikelax** [comment]
basic docker postgres config set up, updated readme, added dotenv for simple config management.

File: api/package.json [hunk 1]
```diff
@@ -14,9 +14,12 @@
...
     "cors": "^2.8.4",
+    "dotenv": "^4.0.0",
     "express": "^4.16.2",
     "express-jwt": "^5.3.0",
     "jwks-rsa": "^1.2.1",
-    "lodash": "^4.17.4"
+    "knex": "^0.14.0",
+    "lodash": "^4.17.4",
+    "pg": "^7.4.0"
   }
```
Comment (nazar) [inline api/package.json:24]: <3

File: README.md [hunk 2]
```diff
@@ -27,10 +27,14 @@ Edit [package.json](./client/package.json) and set `"proxy": "http://localhost:3
...
 
 Install [docker](https://www.docker.com/docker-mac)
+NOTE: Config values for some container parameters (ie. db) are set up using [env files](https://docs.docker.com/compose/environment-variables/).
 
 From the root folder run:
 
 ```shell
+# Pull public postgres docker library
+docker pull postgres
+
```
Comment (nazar) [inline README.md:37]: Interesting that I didn't need the above.

I ran `docker-compose up` and it pulled down the the pg images without me having to `docker pull postgres`

```
react-api-poc (postgres): $ docker-compose up
WARNING: The POSTGRES_PASSWORD variable is not set. Defaulting to a blank string.
Pulling db (postgres:9.6.5-alpine)...
9.6.5-alpine: Pulling from library/postgres
b1f00a6a160c: Pull complete
```

Did that error on your side before `docker pull postgres`?

Comment (mikelax) [reply README.md:37]: Weird yeah I was getting errors about not being able to pull from a public repo.  Finally when i tan the docker pull things started to work.

**Review by nazar — APPROVED** [review]
💯  - I'm in DB heaven!

**Pull request merged**